encryption.go

```diff
@@ -0,0 +1,413 @@
+package winrm
+
+import (
+	"bytes"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"regexp"
+	"strconv"
+	"strings"
+
+	ntlmssp "github.com/CalypsoSys/bobntlmssp"

```

Comment:
Do you know if the upstream has merged your changes?

Reply:
not merged but resolved, so I now use "github.com/bodgit/ntlmssp" directly